Changes.md

```diff
@@ -5,14 +5,30 @@
 HEAD (7.3.0)
 ----------
 
+- **NEW FEATURE** Add `Sidekiq::IterableJob`, iteration support for long-running jobs. [#6286, fatkodima]
+  Iterable jobs are interruptible and can restart quickly if
+  running during a deploy. You must ensure that `each_iteration`
+  doesn't take more than Sidekiq's `-t` timeout (default: 25 seconds). Iterable jobs must not implement `perform`.
+```ruby
+class ProcessArrayJob
+  include Sidekiq::IterableJob
+  def build_enumerator(*args, **kwargs)
+    array_enumerator(args, **kwargs)
+  end
+  def each_iteration(arg)
+    puts arg
+  end
+end
+ProcessArrayJob.perform_async(1, 2, 3)
```

Comment:
This example looks not very much useful and confusing, imo. What about using something like here https://github.com/fatkodima/sidekiq-iteration?tab=readme-ov-file#getting-started ?

Reply:
Yep, I'll remove it there and add 1-2 real world examples to the wiki page tomorrow.